src/Altinn.App.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -176,6 +177,8 @@ IWebHostEnvironment env
         services.Configure<FrontEndSettings>(configuration.GetSection(nameof(FrontEndSettings)));
         services.Configure<PdfGeneratorSettings>(configuration.GetSection(nameof(PdfGeneratorSettings)));
 
+        services.AddLocaltestValidation(configuration);
+
```

Comment:
Should this be wrapped in `env.IsDevelopment()`?

Reply:
Sure 👍